### DISCUSSION TITLE
Are new pull requests still actively monitored?

### DISCUSSION THREAD

**Discussion by ThijmenDam** [serhii-londar/open-source-mac-os-apps#835] [General]
Hi @serhii-londar,

First of all - thanks for the awesome list of software. Helped me out quite a bit. However, I wonder if new applications are still being added, since some pull requests (i.e. for new apps) have been created more than a year ago. I understand you do this in your own time so being patient is not the problem, though I wonder if this repository might be better off if you allow other contributors do do the reviews as well.

**Comment by serhii-londar** [comment]
Hi @ThijmenDam!
I do not have lot enough of free time now to do all the work. So you raised a good point. 
So I can add other contributors to the project. 
So in case you interested in being collaborator of this project please answer this questions:
1. How many apps have you added to the project?
2. Do you have ideas to improve the project?

**Comment by ThijmenDam** [comment]
Hi @serhii-londar, thanks for your quick response and appreciate the effort. To answer your questions:

1. I have created a pull request for one app (#832). Disclaimer: I am the developer of this app.
2. Why not let the approval process be a community effort? Perhaps you can take a look at this GitHub [action](https://github.com/marketplace/actions/git-democracy). You could e.g. automatically approve apps with a certain amount of votes and/or agreement rate. Or, if you still want to have manual control, add the action without automatic approval, and set a threshold of a certain amount of votes that are required to qualify for manual approval (to reduce the amount of apps you have to look into).

**Reply by serhii-londar** [reply]
@ThijmenDam Added you as a collaborator. Feel free to review requests and add new features to the project.

**Reply by ThijmenDam** [reply]
@serhii-londar that's great, thanks! I think I'm available to do some reviews this Sunday.
